Managers/UInputMan.cpp

```diff
@@ -981,4 +1012,74 @@ namespace RTE {
 			}
 		}
 	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifdef __unix__
+	void UInputMan::HandleAllegroMouseInput() {
+		if (_xwin.display == 0)
+			return;
+
+		int nEvents = XPending(_xwin.display);
+		std::vector<XEvent> events;
+		events.reserve(nEvents + 10);
+		for (int i = 0; i < XPending(_xwin.display); ++i) {
```

Comment:
Can you not use nEvents here in the for loop? Or does getting it clear it or something?

Also, it's not a big deal but I'd generally lean towards something slightly more descriptive than nEvents. `pendingEventCount` maybe?